pom.xml

```diff
@@ -0,0 +1,134 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <name>OpenRefine - FAIR Metadata extension</name>
+    <description>Extension for managing metadata in FAIR way</description>
+    <url>https://github.com/FAIRDataTeam/OpenRefine-metadata-extension</url>
+
+    <groupId>org.fair.openrefine</groupId>
```

Comment:
I'll have to discuss what organisation we are using. We do not control the `org.fair` domain for sure, could be `org.gofair` but I'll have to check with @luizbonino.

Reply:
Now it is also related with:

https://github.com/FAIRDataTeam/OpenRefine-metadata-extension/blob/8c5871ba9e3ed2c2e3324327e6da3560dd81c5c3/pom.xml#L14-L17

Reply:
This is the last issue with this PR... Can we let it be as is for now and solve it until the very first release?

Reply:
Yes, for now this is fine. We'll update it to reflect the proper ownership as soon as we can.